scripts/tokenizers/process_asr_text_tokenizer.py

```diff
@@ -137,6 +137,11 @@
     '--spe_control_symbols', default=None, type=str, nargs='+', help='Control symbols for SentencePiece'
 )
 parser.add_argument('--spe_split_digits', action='store_true', help='Split digits into separate tokens.')
+parser.add_argument(
+    '--spe_remove_extra_whitespaces',
```

Comment:
make default value as true

Reply:
as discussed, default this option is false, if one needs to remove_extra_spaces both during training and inference, they will need to specify this flag